client/app/src/services/config.service.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+
+  angular.module('arkclient.services')
+    .service('configService', ['$mdToast', 'gettextCatalog', ConfigService]);
```

Comment:
Looks like some of these services aren't being used

Reply:
Ah yes, I removed the parameters from the constructor, but not here!